src/main/webapp/app/course/dashboards/tutor-course-dashboard/tutor-course-dashboard.component.html

```diff
@@ -4,7 +4,8 @@
             <h2>
                 <fa-icon [icon]="'arrow-left'" (click)="back()" class="back-button mr-2"></fa-icon>
                 <h2 *ngIf="!isExamMode">{{ 'artemisApp.tutorCourseDashboard.pageHeader' | translate }} {{ course?.title }}</h2>
-                <h2 *ngIf="isExamMode">{{ 'artemisApp.tutorCourseDashboard.pageHeaderExam' | translate }} {{ exam?.title }}</h2>
+                <h2 *ngIf="isExamMode && !isTestRun">{{ 'artemisApp.tutorCourseDashboard.pageHeaderExam' | translate }} {{ exam?.title }}</h2>
+                <h2 *ngIf="isExamMode && isTestRun">{{ 'artemisApp.tutorCourseDashboard.pageHeaderExamTestRun' | translate }} {{ exam?.title }}</h2>
```

Comment:
You only need to switch the translation string conditionally.

Reply:
488d9189189d5a4ef48ea0717b95cd9fe6656810